Maddori.Apple/Maddori.Apple/Global/UIComponent/LabelButtonView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Label+ButtonView.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2022/10/22.
+//
+
+import UIKit
+
+import SnapKit
+
+class LabelButtonView: UIView {
+    
+    var buttonAction: (() -> ())?
+    
+    var subText: String = "" {
+        didSet {
+            subLabel.text = subText
+        }
+    }
+    
+    var subButtonText: String = "" {
+        didSet {
+            subbutton.setTitle(subButtonText, for: .normal)
+        }
+    }
+    
+    // MARK: - property
+    private lazy var subbutton: UIButton = {
```

Comment:
p4;

```suggestion
    // MARK: - property
    
    private lazy var subButton: UIButton = {
```

혹쉬 한줄만 띄고, subButton으로 변경해 주실 수 있나요오...?? ㅎㅏ하하하하

Reply:
[443fa3a](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/32/commits/443fa3aab12aed8cbac1058e6cd895e0017948f0)